tests/test_gitlab.py

```diff
@@ -54,7 +58,7 @@ class TestGitlabIssue(AbstractServiceTest, ServiceTest):
             "id": 1,
             "title": "v1.0",
             "description": "",
-            "due_date": "2012-07-20",
+            "due_date": arbitrary_duedate.date().isoformat(),
```

Comment:
Well, the actual format doesn't have the time associated with it… I guess the internal "it's a date, just put the time on it too" logic is doing this. Taskwarrior only has a single "datetime" type, so I'm not sure how best to fix this.